Project.toml

```diff
@@ -16,33 +16,49 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [weakdeps]
+AxisKeys = "94b1ba4f-4ee9-5380-92f1-94cde586c3c5"
+IntervalSets = "8197267c-284f-5f27-9208-e0e47529a953"
+SkyCoords = "fc659fc5-75a3-5475-a2ea-3da92c065361"
```

Comment:
The other packages seem to be "general purpose" computing packages, but SkyCoords seems very niche?